website/docs/reference/global-cli-flags.md

```diff
@@ -3,15 +3,15 @@ title: "Global CLI flags"
 id: "global-cli-flags"
 ---
 
-dbt commands (eg. `run`, `test`, etc) support their own command-specific CLI flags. In addition to these command specific flags, the dbt command itself supports "global" flags applicable to *all* subcommands.
+dbt commands, such as `run` or `test`, support their own command-specific CLI flags. In addition, the dbt command itself supports "global" flags applicable to *all* subcommands.
 
-As of v1.0, the vast majority of global CLI flags map to [global configs](global-configs), which can also be configured via environment variables or `dbt_project.yml`.
+As of v1.0, the vast majority of global CLI flags map to [global configs](global-configs), which can also be configured via environment variables or in the `dbt_project.yml`.
```

Comment:
@jtcohen6 Should this be `profiles.yml` not dbt_project.yml?

Reply:
Yes, it should!